api/src/main/java/com/community/api/endpoint/avisoft/controller/Account/AccountEndPoint.java

```diff
@@ -8,6 +8,7 @@
 import com.community.api.services.*;
```

Comment:
Not import using *
only import the required classes.
![image](https://github.com/user-attachments/assets/30cffd67-63e4-4874-abc1-5cb49beb1172)